src/screens/Processors/ProcessorCards.res

```diff
@@ -46,6 +46,7 @@ let make = (
   ~showTestProcessor: bool,
   ~urlPrefix: string,
   ~connectorType=ConnectorTypes.Processor,
+  ~setProcessorModal: (bool => bool) => unit=_ => (),
```

Comment:
Why this type added? Can be removed i guess it will automatically check that. I guess check once via removing